packages/components/pie-card-container/src/index.ts

```diff
@@ -31,19 +31,23 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
     @property({ type: Object })
     public aria!: AriaProps;
 
+    @property({ type: Boolean })
+    public isDraggable = false;
+
     render () {
         const {
             href,
             target,
             rel,
             disabled,
             variant,
+            isDraggable,
         } = this;
 
         return html`
                 <div
                     variant=${variant}
-                    class="c-card-container"
+                    class="c-card-container ${isDraggable ? 'isDraggable' : ''}"
```

Comment:
do you think we can use HTML attributes similar to the disabled state for this case just to be consistent with the rest of the implementions